spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -5865,6 +5865,53 @@ disable transaction management for a test or for the whole class as follows:
 A list of the auto-configuration that is enabled by `@DataNeo4jTest` can be
 <<appendix-test-auto-configuration#test-auto-configuration,found in the appendix>>.
 
+[[boot-features-testing-spring-boot-applications-testing-autoconfigured-redis-test]]
+==== Auto-configured Data Redis tests
+`@DataRedisTest` can be used if you want to test Redis applications.
+
+[source,java,indent=0]
+----
+	import org.junit.runner.RunWith;
+	import org.springframework.beans.factory.annotation.Autowired;
+	import org.springframework.boot.test.autoconfigure.data.redis.DataRedisTest;
+	import org.springframework.test.context.junit4.SpringRunner;
+
+	@RunWith(SpringRunner.class)
+	@DataRedisTest
+	public class ExampleDataRedisTests {
+
+		@Autowired
+		private YourRepository repository;
+
+		//
+	}
+----
+
+Data Redis tests are transactional and rollback at the end of each test by default,
```

Comment:
By default, `RedisTemplate` is not configured with `setEnableTransactionSupport(true)` so there's no transaction support out of the box.